addons/hr_expense/static/src/views/list.js

```diff
@@ -28,27 +34,28 @@ export class ExpenseListController extends ExpenseDocumentUpload(ListController)
 
     displaySubmit() {
         const records = this.model.root.selection;
-        return records.length && records.every(record => record.data.state === 'draft') && this.isExpenseSheet;
+        return records.length && records.every(record => record.data.state === 'draft') && this.isExpenseSheet && !this.onMobile;
```

Comment:
I need to hide these buttons on mobile using these methods, which are passed to `t-if` directives, in order to make them completely hidden on mobile. The approach followed before was to hide them using classes `d-none d-md-flex` which works fine, but has an issue. The issue is that the DOM acts as if these buttons exist, and it shows an arrow on mobile that when clicked should show the buttons, but when clicked, it shows nothing because the buttons are hidden. Laura complained about that, and that was my approach of fixing it.